packages/nimble-components/src/rich-text/editor/models/create-tiptap-editor.ts

```diff
@@ -216,7 +216,14 @@ function createCustomMentionExtension(
     }).configure({
         suggestion: {
             char: config.character,
-            decorationTag: config.viewElement,
+            /**
```

Comment:
Based on the other comment I have a general concern that the rich-text-editor and the rich-text-viewer have too much knowledge of the user mention implementation.

for example:
rich-text\editor\testing\rich-text-editor.pageobject.ts
rich-text\editor\testing\rich-text-editor-utils.ts
rich-text\editor\testing\types.ts

It should be organized so that anything specific to user mentions are handled from those folders instead (rich-text-mention\users\testing\*), the rich-text\ directory should not have any dependencies on specific mention implementations. **We should be able to completely delete user mentions without it impacting tests \ implementation for rich-text-viewer and rich-text-editor**.

The parts related to this PR are specifically styling changes. So this concern could be captured as a separate tech debt issue and addressed separately.

Reply:
Created a tech debt and will be taken up in the follow-up PR: https://github.com/ni/nimble/issues/1724